src/main/resources/localstack/s3/core/sites/sites.json

```diff
@@ -20,7 +20,8 @@
     "enabled": true,
     "clientTypes": ["PUBLISHER"],
     "visible": true,
-    "domain_names": ["example.com"]
+    "domain_names": ["example.com"],
+    "app_names": ["com.123.game.app.android", "123456789", "com.123.game.app.ios"]
```

Comment:
for future testing i suggest to add some case sensitiveness into these values, e.g. 

com.123.Game.App.android
com.123.Game.App.ios

Reply:
I adjusted the value to include a mixture of lower and upper cases.